.vscode/launch.json

```diff
@@ -6,12 +6,20 @@
 	"version": "0.2.0",
     "configurations": [
         {
-            "name": "Extension",
             "type": "extensionHost",
             "request": "launch",
+            "name": "Launch Client",
+            "runtimeExecutable": "${execPath}",
             "args": [
-                "--extensionDevelopmentPath=${workspaceFolder}"
-            ]
+                "--extensionDevelopmentPath=${workspaceRoot}"
+            ],
+            "outFiles": [
+                "${workspaceRoot}/out/**/*.js"
+            ],
+            "preLaunchTask": {
```

Comment:
Why is this needed?

Reply:
This will watch for changes and recompile the typescript when you test the extension from in VSCode (using F5).